yarn.lock

```diff
@@ -12190,6 +12235,11 @@ ses@^0.18.0:
   resolved "https://registry.yarnpkg.com/ses/-/ses-0.18.0.tgz#9bc1f17712e2f2cf9a38c83156197a6fe722822e"
   integrity sha512-WMEOltdIqYk11uZiD+0ANnTLcpAQXyLAPm5NneJSbSPEzxhSn4G7B/bs3dtSOcEicAlFI6WFXqXKDMu9BBCj+w==
 
+ses@^0.18.3:
```

Comment:
this adds a ses but doesn't take one away. `npx yarn-deduplicate` would clean it up. When I ran it 297 lines were removed. Many of them were dupes of Endo deps so I think this is worth doing.